datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -1,7 +1,8 @@
 adodbapi==2.6.0.7; sys_platform == 'win32'
 asn1crypto==0.24.0
 beautifulsoup4==4.5.1
-boto==2.46.1
+boto==2.46.1; python_version < '3.0'
+boto3==1.9.67; python_version > '3.0'
```

Comment:
Let's do all of these dep changes first in a separate PR

Reply:
Actually no other check uses boto, so disregard this

Reply:
yea I checked that before doing this

Reply:
boto _is_ used by a5 so we'll need to ensure it doesn't break a5 but it should be fine